molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/resources/impl/MultiResourceConfigImpl.java

```diff
@@ -18,18 +18,36 @@
 	private String chromosomesProperty;
 	private String filePatternProperty;
 	private String folderProperty;
+	private Map<String, String> overrideChromosomeFiles;
 
 	public static final String DEFAULT_CHROMOSOMES = "1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,Y,X";
 	public static final String DEFAULT_PATTERN = "chr%.vcf";
 	public static final String DEFAULT_FOLDER = "/data/resources/";
 
+	/**
+	 * overrideChromosomeFiles contains the chromosome that do not comply with the pattern.
+	 * those chromosomes and corresponding filenames should be stated comma separated per chromosome,
+	 * and the chromosome and filename itself should be colon separated.
+	 * e.g. 1:file1.vcf,2:file2.txt,X:XYZ.vcf
+	 * */
 	public MultiResourceConfigImpl(String chromosomesProperty, String filePatternProperty, String folderProperty,
-			MolgenisSettings molgenisSettings)
+			MolgenisSettings molgenisSettings, String overrideChromosomeFiles)
```

Comment:
Should be updated to using the property here instead of the value, I will update